doc/examples/examples.rst

```diff
@@ -73,8 +73,6 @@ Configuration
    planning_adapters/planning_adapters_tutorial.rst
    persistent_scenes_and_states/persistent_scenes_and_states
 
-**Note:** The list of the robots above are for MoveIt, a list is on the works for MoveIt 2.
```

Comment:
This sentence was in the wrong place and generally just feels like it wasn't really needed.